content/en/blog/falcosidekick-2-27-0-ui-2-1-0/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Falcosidekick 2.27.0 and Falcosidekick-UI 2.1.0
+linktitle: Falcosidekick 2.27.0 and Falcosidekick-UI 2.1.0
+date: 2022-12-16
+author: Thomas Labarussias
+slug: falcosidekick-2-27-0-ui-2-1-0
+images:
+  - images/falcosidekick-featured.png
+---
+
+![](images/falcosidekick-featured.png)
+
+So much good things happened for Falcosidekick and Falcosidekick UI this year, it's still incredible such projects became so beloved and useful for the community. To all contributors, promotors and users, a big big thank you.
+
+Christmas is coming, it's the time to release new versions of and reach in next January, 10 millions of Docker pulls for Falcsidekick.
```

Comment:
Christmas is coming. It's time to release new versions! The goal is to reach 10 million Docker pulls for Falcosidekick this coming January 2023.